### ISSUE TITLE
missing alt text for "cloudinary tag images"

### ISSUE THREAD

**Issue by rockroman** [rockroman/CI_PP4-Knowledge-Flow#41]
**Describe the bug**
Cloudinary images rendered with a cloudinary template tags are missing alt text

**Solution**
after quick research, I decided to use Jquery and add alt text to images by setting a desired class for the image and then targeting that class to add alternative text
